spec/javascripts/helpers/jasmine_examples/SpecHelper.js

```diff
@@ -0,0 +1,15 @@
+beforeEach(function () {
```

Comment:
Oh, weird - when I tried to run the tests in the browser, it complained that it couldn't find the custom matcher, which was why I wrote my own implementation in PlayerSpec.js.